src-electron/importexport/import-isc.js

```diff
@@ -638,7 +638,7 @@ async function loadSessionKeyValues(db, sessionId, keyValues) {
  * @param {*} state
  * @param {*} sessionId
  */
-async function iscDataLoader(db, state, sessionId) {
+async function iscDataLoader(db, state, sessionId, packageMatch) {
```

Comment:
Is packageMatch being used here or can it be removed?

Reply:
Can be removed. I added it, because the import.js will pass it to this function, as it's using a common interface to both import-isc.js and import-json.js. It wasn't clear to me which is better:
   1.) Declare an argument that IS passed and then not use it.
   2.) Pretend that there is no argument, even though the caller WILL add the argument.

Reply:
Can the interface not have this?

Reply:
I'll just remove it from here...